content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -149,12 +149,12 @@ users:
     username: exp
 ```
 
-The `fake-ca-file`, `fake-cert-file` and `fake-key-file` above is the placeholders
-for the real path of the certification files. You need change these to the real path
-of certification files in your environment.
+The `fake-ca-file`, `fake-cert-file` and `fake-key-file` above are the placeholders
+for the pathnames of the certificate files. You need change these to the actual pathnames
+of certificate files in your environment.
 
-Some times you may want to use base64 encoded data here instead of the path of the 
-certification files, then you need add the suffix `-data` to the keys. For example,
+Sometimes you may want to use Base64-encoded data embeddedhere instead of separate
```

Comment:
I think this has a tiny regression:
```suggestion
Sometimes you may want to use Base64-encoded data embedded here, instead of separate
```
?

Reply:
Yep. Fixed.